apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/ReleaseHistoryService.java

```diff
@@ -90,6 +101,22 @@ public ReleaseHistoryBO findLatestByPreviousReleaseIdAndOperation(Env env, long
       bos.add(transformReleaseHistoryDTO2BO(dto, release));
     }
 
+    Set<String> operatorIdSet = bos.stream()
```

Comment:
I think we could first `additionalUserInfoEnrichService.enrichAdditionalUserInfo(releases)` and then change the `transformReleaseHistoryDTO2BO` logic with `bo.setOperatorPreferredUsername(dto.getDataChangeCreatedByPreferredUsername());` so that we don't need to expose the enrich logic here?